sechub-cli/src/daimler.com/sechub/util/filepathmatcher.go

```diff
@@ -3,91 +3,38 @@ package util
 
 import (
 	"fmt"
-	"path/filepath"
+	"regexp"
 	"strings"
 )
 
-//
-// This method provides ANT like selectors.
+// Filepathmatch - This method provides ANT like selectors.
+// See https://ant.apache.org/manual/dirtasks.html
 // For example: "**/a*.txt" will accept
-//               - "/home/tester/xyz/a1234.txt
-//               - "/root/a1b.txt
+//               - "/home/tester/xyz/a1234.txt"
+//               - "a1b.txt"
 //
 func Filepathmatch(path string, pattern string) (result bool) {
-	/* ------------------------------ */
-	/* Explanation - how it works ... */
-	/* ------------------------------ */
-	// - we separate pattern into different path patterns by path wildcards into elements
-	// - then the last element is inspected to be a file wildcard
-	// - file wildcards are resolved /tried by golang filepath.Match method
-	// - if this is okay we do our special path wildcard matching
-
-
-	/* separate patterns containing path wildcards */
-	subPattern := strings.Split(pattern, "**/")
-//	fmt.Printf("\n%s - seperated into %q", pattern, subPattern)
-	length := len(subPattern)
-	potentialFilePattern := subPattern[length-1]
 
-	/* ------------------------ */
-	/* handle file name pattern */
```

Comment:
@sven-dmlr : Thank you for removing my old code - your solution is shorter, better and works also in deep 👍